src/ui.container-accordion/UAccordion.vue

```diff
@@ -42,17 +45,17 @@ const {
 } = useAttrs(props, { isOpened });
 
 const toggleIcon = computed(() =>
-  isOpened.value ? config.value.defaults.collapseIcon : config.value.defaults.expandIcon,
+  isOpened.value ? config.value?.defaults?.collapseIcon : config.value?.defaults?.expandIcon,
 );
 
-const dividerSize = computed(() => {
-  const sizes = {
+const dividerSize = computed((): DividerSize => {
+  const sizes: Record<AccordionSize, DividerSize> = {
     sm: "md",
     md: "lg",
     lg: "xl",
   };
 
-  return sizes[props.size];
+  return sizes[props.size as AccordionSize];
```

Comment:
this types in no needed in computed at all, should work without it, same as in `UButton` for example.